cogs5e/sheetManager.py

```diff
@@ -385,6 +385,66 @@ async def character(self, ctx, *, name: str = None):
 
         await ctx.send(f"Active character changed to {char.name}.", delete_after=20)
 
+    @character.command(name='server')
+    async def server_character(self, ctx, *, name: str = None):
```

Comment:
Instead of having a command signature of `!char server <name>`, consider having `!char server` simply set/unset the current globally active character as server active. This way, you could remove the `!char unset` command, which feels somewhat unclear.

Btw, naming convention for subcommands is `async def (parent)_(name)`, so in this case, `async def character_server`.

Reply:
I feel like the ability to set the server character without changing the global character kinda bogs down on the point of all this. I do feel however that these commands could possibly be unified somewhat.